pkg/cli/connect_helm.go

```diff
@@ -88,8 +88,17 @@ func ConnectHelm(ctx context.Context, options *ConnectOptions, globalFlags *flag
 }
 
 func (cmd *connectHelm) connect(ctx context.Context, vCluster *find.VCluster, command []string) error {
+	currentContext, _, err := find.CurrentContext()
+	if err != nil {
+		return err
+	}
+	if currentContext == find.VClusterContextName(vCluster.Name, vCluster.Namespace, vCluster.Context) {
+		cmd.Log.Infof("already connected to vcluster %q", vCluster.Name)
```

Comment:
Can we somehow check if the vCluster is reachable here like we do below so that we reconnect if we cannot reach the vCluster?